### ISSUE TITLE
`asciidoc` tool produces issues, we should only support `asciidoctor` 

### ISSUE THREAD

**Issue by ghost** [openSUSE/daps#486]
As `asciidoc` is not properly maintained anymore, there is no real reason to keep it supported anymore... especially given that it produces additional errors that are not an issue when using asciidoctor.

E.g. in the CAP release notes repo, `asciidoc` produces XML with errors such as the following:
```
[REPO]/rn-cap/build/.adoc/release-notes.xml:1302: parser error : Opening and ending tag mismatch: section line 1273 and programlisting
-service-broker usb broker-admin "$USB_PASSWORD" "$USB_ENDPOINT"</programlisting
                                                                               ^
[REPO]/rn-cap/build/.adoc/release-notes.xml:1595: parser error : Opening and ending tag mismatch: listitem line 1590 and programlisting
kube.external_ips=["1.1.1.1", "2.2.2.2"]</programlisting>
                                                         ^
[REPO]/rn-cap/build/.adoc/release-notes.xml:1596: parser error : Opening and ending tag mismatch: itemizedlist line 1557 and listitem
</listitem>
           ^
[...]
```
`asciidoctor` on the other hand builds the same repo properly.

**Comment by ghost** [comment]
The feature set of `asciidoc` and `asciidoctor` is also not the same -- I have noticed e.g.:

* `asciidoc` supports 16-color text, `asciidoctor` does not
* `asciidoc` supports `...-docinfo.xml` files that supply information into the `info` element of a book, `asciidoctor` does not
*  `asciidoc` generates DocBook 4, `asciidoctor` generates DocBook 5

This is somewhere between confusing and infuriating.

**Comment by fsundermeyer** [comment]
Fixed with 4d3bcbf9b7601aa0dc52370c0c44c541cd9ed888
